### ISSUE TITLE
Persist ip_conntrack after reboot

### ISSUE THREAD

**Issue by vaubarth** [elastic/ansible-elastic-cloud-enterprise#29]
`ip_conntrack` is currently loaded via ansible `modprobe` which does not persist it anywhere.

We need to create a `/etc/modules-load.d/` file in a seperate task (e.g. `ip_conntrack.conf`)

**Comment by selste** [comment]
One of our Admins dug around a little bit and found out that dockerd forces the ip_conntrack module to be loaded (on our RHEL 7.6 systems) if necessary.
I've added a separate YAML file to the RHEL-7 folder called 'kernel_modules.yml':
```
---
- name: Enable kernel modules
  template:
   src: templates/module.conf.j2
   dest: "/etc/modules-load.d/{{ item.name }}.conf"
  with_items:
  - { name: 'conntrack', kernel_module: 'ip_conntrack' }
  - { name: 'overlay', kernel_module: 'overlay' }
```
and - of course - a simple template 'module.conf.j2' consisting of:
```
{{ item.kernel_module }}
```
It's reasonably generic, although there is a redundancy: the overlay kernel module is also defined in 'vars/os_[Distro].yml'.

**Comment by vaubarth** [comment]
:+1: 
I think we should make the ip_contrack module generic for all distros and include it in `tasks/system/general/sysctl_scripts.yml`
Then we can seperate the `overlay` module and only include that where it is necesarry.
What do you think?

**Comment by selste** [comment]
Generally, yes.
I had a quick look at a couple of different Linux installations at hand (Linux Mint, Ubuntu 18.04, CentOS 7 and RHEL 7) and none of them had either of the modules included per default - although these are manual installations, a bit different from what'd be used in a company.
If both modules are (generally) needed, but handled separately ... that sounds like violating the DRY principle to me?!
Bundling this task up into a distinct script might be a better approach - but i really don't have enough experience using Ansible for an assessment.
Summing up - i'm perfectly fine with both solutions. In the end it's your project and your call

**Comment by vaubarth** [comment]
Sorry for the delay. This is now solved by one tasks as you suggested that works slightly differently but does the same thing.
